pkg/internal/config/config_test.go

```diff
@@ -68,7 +70,8 @@ func TestNewShouldApplyDefaultsWhenOptionalFieldsAreMissing(t *testing.T) {
 		NetworkAttachmentDefinitionName: networkAttachmentDefinitionName,
 		TrafficGeneratorPacketsPerSecondInMillions: config.TrafficGeneratorPacketsPerSecondInMillionsDefault,
 		PortBandwidthGB:                config.PortBandwidthGBDefault,
-		TrafficGeneratorEastMacAddress: trafficGeneratorMacAddressDefault,
+		TrafficGeneratorEastMacAddress: trafficGeneratorEastMacAddressDefault,
```

Comment:
I think this change should belong to the previous commit.

Reply:
totally.